PVList.md

```diff
@@ -0,0 +1,364 @@
+# IPMI Manager Process Variables naming list
+
+This document has the purpose of making easier to understand the PV
+substitution tree in this IOC and to follow LNLS's naming convention. It is
+also useful for any user to check which PVs are available.
+
+Each new record created, or aliased from the ipmiComm module, must be
+documented here.
+
+```
+AREA=IA-{{01..20}RaBPM|20RaBPMTL} # Area = LNLS uTCA crates
+DIS=CO  #Discipline = Control
+```
+## MCH
+### `$(AREA):$(DIS)-MCH_Base:`
+<details>
```

Comment:
```suggestion
## MCH

### `$(AREA):$(DIS)-MCH_Base:`

<details>
```

Reminder that Markdown can also be read raw, so whitespace is a helpful addition. Please fix it in other spots as well